docs/docs/operator/continuous-scanning.md

```diff
@@ -0,0 +1,104 @@
+# Continuous Scanning
+
+To let cluster operators see the current security posture of their cluster, Kubescape provides the Continuous Scanning feature. Once enabled, Kubescape will constantly monitor the cluster for changes, evaluate their impact on the overall cluster security and reflect its findings in the cluster-, namespace- and workload-scoped security reports. Essentially, this means you'll always get the latest scoop on your cluster’s security!
+
+## Installation
+
+Continuous Scanning is built into the Kubescape Operator Helm chart. To use this capibility, you only need to enable it. Start by navigating to the `values.yaml` file and make sure that the corresponding `capabilities.continuousScan` key is set to `enabled`, like so:
+
+```yaml
+capabilities:
+  continuousScan: enable  # Make sure this is set to "enable"
+```
+
+Once you apply the chart with the capability enabled, Kubescape will continuously secure your cluster and provide the scan results as Custom Resources.
+
+## Accessing Results
+
+### For the Whole Cluster
+
+Kubescape provides scan results as Custom Resources so you can access them in the same convenient way you access other Kubernetes objects. Let’s assume you’d like to see a birds-eye view of you cluster’s security. In Kubescape terms, that would mean taking a look at the cluster-level configuration scan summary:
+
+```
+kubectl get workloadconfigurationscansummaries -o yaml
+```
+
+Running this command will return you a YAML-formatted list of configuration scan summaries for your cluster by namespaces.
+
+On clusters with many namespaces, the results might be overwhelming and might even exceed your terminal history. Since Kubescape serves results as Kubernetes objects, which are YAML files at its core, you can do your usual tricks: pipe them to files, text editors etc. A trick we commonly use is:
```

Comment:
```suggestion
On clusters with many namespaces, the results might be overwhelming and might even exceed your terminal history. Since Kubescape serves results as Kubernetes objects, which are YAML files at the end of the day, you can apply your usual processes to aggregate them in a readable way. For example - pipe them to files, text editors etc. We commonly use the following command:
```

Reply:
Fixed.